spec/controllers/stories_bulk_update_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+describe StoriesBulkUpdateController do
+  describe '#create' do
+    def catch_column(model, arg)
```

Comment:
Can we rename it to `map_to_attribute`? The word `catch` has a special meaning on programming so it may be confusing.

Reply:
Why didn't I think in this method name?! Very good, thank you!